playersScore.js

```diff
@@ -0,0 +1,131 @@
+const player1Name = document.getElementById("p1-input");
+const player2Name = document.getElementById("p2-input");
+const startBtn = document.getElementById("start-btn");
+const playersSection = document.getElementById("players-container");
+const scoreContainer = document.getElementById("score");
+const restartBtn = document.getElementById("restart-btn");
+
+startBtn.addEventListener("click", startQuiz);
+// Enabling of Start button
+playersSection.addEventListener("change", () => {
+	if (player1Name.value && player2Name.value) {
+		startBtn.disabled = false;
```

Comment:
Nice safeguard to prevent the game from being started early.

Reply:
However, you do not handle the `else` case, meaning that one of the player names is empty. But why would that be a problem? Well, a fun way to exploit this is that I can enter names for both players, which will make the Start button become active, and then remove one of the names, and the start button does not deactivate. This means I **can** start a game with empty player name(s). 😉

Reply:
Thus, in the `else` case you would set `startBtn.disabled = true;`